hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/distributions/smtable/SwModuleTable.java

```diff
@@ -369,30 +323,23 @@ private String getNameAndVerion(final Object itemId) {
         return name + "." + version;
```

Comment:
Yes, you're right. Compiler is doing this. I found this additional post which explains it quite well => http://stackoverflow.com/a/14927864

Only in loops you have to think about introducing StringBuilder manually.